FirebaseAuthUI/Strings/en.lproj/FirebaseAuthUI.strings

```diff
@@ -25,6 +25,9 @@
 /* Password text field placeholder. */
 "EnterYourPassword" = "Enter your password";
 
+/* Error message displayed when user enters an empty passwrd. */
+"InvalidPasswordError" = "Password can not be empty.";
```

Comment:
both forms are valid https://en.oxforddictionaries.com/usage/cannot-or-can-not
But I'll fix to preferred one